hail/src/main/scala/is/hail/expr/ir/BlockMatrixIR.scala

```diff
@@ -112,7 +112,7 @@ class BlockMatrixLiteral(value: BlockMatrix) extends BlockMatrixIR {
     BlockMatrixType(TFloat64(), shape, isRowVector, value.blockSize)
   }
 
-  override def children: IndexedSeq[BaseIR] = Array.empty[BlockMatrixIR]
+  @transient lazy val children: IndexedSeq[BaseIR] = Array.empty[BlockMatrixIR]
```

Comment:
Why is this @transient?  We shouldn't be serializing IRs.

Reply:
oh, okay. will remove transient